tests/test_language_intents.py

```diff
@@ -33,10 +45,23 @@ def test_language_common(
     ), "_common.yaml is a common file and should not contain intents"
 
 
-def test_language_sentences(
-    language_sentences: Intents, intent_schemas: Dict[str, Any]
+def do_test_language_sentences(
+    file_name: str,
+    intent_schemas: Dict[str, Any],
+    language_sentences_yaml: Dict[str, Any],
+    common_language_intents: Intents,
 ):
     """Ensure all language sentences contain valid slots, lists, rules, etc."""
+    parsed_sentences_without_common = Intents.from_dict(
+        language_sentences_yaml[f"{file_name}.yaml"]
+    )
+
+    # Merge common rules with file specific intents.
+    language_sentences = dataclasses.replace(
+        common_language_intents, intents=parsed_sentences_without_common.intents
+    )
```

Comment:
@synesthesiam instead of relying on `Intents.from_dict` I now manually combine the results of 2. The sentence file only contains `intents` while `_common.yaml` contains the other fields. Is that ok?